CHANGES.rst

```diff
@@ -655,6 +655,9 @@ Bug Fixes
     (eg. ``NDDataArray`` from ``NDData``) now sets the attributes other than
     ``data`` as it should [#4137].
 
+  - ``NDDataBase`` now tries to set the ``parent_nddata`` attribute of the
+    uncertainty.
```

Comment:
Could you please add the pull request number in brackets at the end of the changelog entry?